drivers/adc-dac/ad5592r/ad5592r-base.h

```diff
@@ -77,11 +77,42 @@ enum ad5592r_registers {
 	AD5592R_REG_RESET		= 0xF,
 };
 
+#define AD5592R_REG_PD_PD_ALL		BIT(10)
+#define AD5592R_REG_PD_PD_ALL_RD(x)		(((x) & AD5592R_REG_PD_PD_ALL) >> 10)
 #define AD5592R_REG_PD_EN_REF		BIT(9)
-#define AD5592R_REG_CTRL_ADC_RANGE	BIT(5)
```

Comment:
the commit refers to adding macros, why are these removed ?

Reply:
The macros from ADC and DAC range have just been moved. You can see them on line 95 and 97